packages/ui-react/src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -107,72 +79,82 @@ const ChooseOfferForm: React.FC<Props> = ({
     </div>
   );
 
-  const renderOfferBox = (offer: Offer) => {
-    if (isSVODOffer(offer)) {
-      const isMonthly = offer.period === 'month';
+  if (isSVODOffer(offer)) {
+    const isMonthly = offer.period === 'month';
 
-      return (
-        <OfferBox
-          offer={offer}
-          key={offer.offerId}
-          title={isMonthly ? t('choose_offer.monthly') : t('choose_offer.yearly')}
-          ariaLabel={isMonthly ? t('choose_offer.monthly_subscription') : t('choose_offer.yearly_subscription')}
-          secondBenefit={t('choose_offer.benefits.cancel_anytime')}
-          periodString={isMonthly ? t('periods.month') : t('periods.year')}
-        />
-      );
-    }
+    return renderOption({
+      title: isMonthly ? t('choose_offer.monthly') : t('choose_offer.yearly'),
+      secondBenefit: t('choose_offer.benefits.cancel_anytime'),
+      periodString: isMonthly ? t('periods.month') : t('periods.year'),
+    });
```

Comment:
This seems to revert an a11y fix.

Reply:
No, it re-does one. This is because I had already rebased on the release branch, and then tried to undo it.